Makefile

```diff
@@ -59,6 +59,12 @@ gqlgen:
 protopb:
 	protoc -I=. --go_out=plugins=grpc:.  pkg/apis/cni/v1alpha1/cni.proto
 
+apidocs-gen:
+	go install github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0
```

Comment:
maybe there is no need to install it each time we generate code.

Reply:
maybe we should check it before install.